hbase-server/src/test/java/org/apache/hadoop/hbase/master/balancer/TestStochasticLoadBalancer.java

```diff
@@ -53,6 +47,10 @@
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 
+import static org.junit.Assert.*;
```

Comment:
Am getting the following checkstyle issues, when running checkstyle locally:

Error | imports | AvoidStarImport | Using the '.*' form of import should be avoided - org.junit.Assert.*. | 50
-- | -- | -- | -- | --
Error | imports | ImportOrder | Wrong order for 'org.junit.Assert.*' import. | 50

Reply:
I'm sorry for the trouble, I must admit that I was trusting the import style of my IDE 😄 

Will fix it in another commit that will be checked locally this time